packages/ic-response-verification/src/verification/verify_request_response_pair.rs

```diff
@@ -46,11 +47,11 @@ pub fn verify_request_response_pair(
         .get("content-encoding")
         .map(|encoding| encoding.as_str());
 
-    let Some(certificate_header) = headers.get("ic-certificate") else {
+    let Some(certificate_header_str) = headers.get("ic-certificate") else {
```

Comment:
nit: since we now import `CERTIFICATE_HEADER_NAME`, maybe use `CERTIFICATE_HEADER_NAME.to_lowercase()` instead of `"ic-certificate"`?